packages/patternfly-4/react-core/src/components/Wizard/WizardContext.ts

```diff
@@ -1,5 +1,46 @@
 import * as React from 'react';
 
-const WizardContext = React.createContext({});
+export interface WizardStep {
+  /** Optional identifier */
+  id?: string | number;
+  /** The name of the step */
+  name: string;
+  /** The component to render in the main body */
+  component?: any;
+  /** Setting to true hides the side nav and footer */
+  isFinishedStep?: boolean;
+  /** Enables or disables the step in the navigation. Enabled by default. */
+  canJumpTo?: boolean;
+  /** Sub steps */
+  steps?: WizardStep[];
+  /** (Unused if footer is controlled) Can change the Next button text. If nextButtonText is also set for the Wizard, this step specific one overrides it. */
+  nextButtonText?: string;
+  /** (Unused if footer is controlled) The condition needed to enable the Next button */
+  enableNext?: boolean;
+  /** (Unused if footer is controlled) True to hide the Cancel button */
+  hideCancelButton?: boolean;
+  /** (Unused if footer is controlled) True to hide the Back button */
+  hideBackButton?: boolean;
+}
```

Comment:
I don't know that we can move the WizardStep interface without introducing a breaking change. Won't this change where WizardStep is imported from?

Reply:
Users of this lib import it from the top-level [index.ts](https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-core/src/index.ts). 
e.g.
```ts
import {
    WizardContextConsumer,
    WizardStep
} from '@patternfly/react-core';
```

Since that's the document way to import stuff from react-core, i think is not a breaking change :). - Also, after re-reading the docs, I think this `WizardStep` is not even documented (I didn't find it [here](https://patternfly-react.surge.sh/patternfly-4/documentation/react/components/wizard), only references that it exists in some types) - Is there a `typescript` version of these docs?

Reply:
Lack of doc wouldn't negate a breaking change. However, as long as it doesn't change the import path, I'm fine with it.

Reply:
this interface is not context specific, can we leave this in Wizard.tsx? Doesn't have to be moved